build/BUILD.bazel

```diff
@@ -90,13 +96,27 @@ wasmx_vm_cmd = select({
     "//conditions:default": "",
 })
 
+wasmx_load_module = select({
```

Comment:
What's stopping us from just adding this to the template?

```
[...]

error_log ${{PROXY_ERROR_LOG}} ${{LOG_LEVEL}};

> if wasm then
load_module /path/to/ngx_wasm_module.so;
> end

[...]
```

Reply:
If the module is built statically, `ngx_wasm_module.so` will not exist and the config check will fail. We can remove the line in this case instead of adding, but it seems the build script complexity would be similar.

Reply:
Hmm. I bet there's a way to detect this in the conf loader.

We could parse [ngx.config.nginx_configure](https://github.com/openresty/lua-nginx-module#ngxconfignginx_configure) to check for `--add-module` vs `--add-dynamic-module` and choose whether or not to add `load_module`. This would even help to provide nicer errors before doing a conf test (i.e. if the user turns wasm on, but we can't find ngx_wasm_module in the configure string, raise an error).

Either that or maybe we copy the dynamic module to some known path, and the conf loader can check for the existence of /path/to/dynamic/ngx_wasm_module.so in order to determine adding `load_module`.

Reply:
That's a nice idea! I made the needed changes, wdyt?

Now the module is looked up in 3 different locations, in order: `KONG_DYNAMIC_MODULES_PATH` env var, the build prefix path and then `/usr/local/openresty/nginx/modules`. The second one is needed for the dev env but will not work in the production env. I think it's a acceptable performance penalty as this function is called only when starting.